load-test-framework/src/main/java/com/google/pubsub/flic/Driver.java

```diff
@@ -141,6 +154,22 @@
           "message loss. If set less than 1, this flag is ignored."
   )
   private int numberOfMessages = 0;
+  @Parameter(
+      names = {"--max_publish_latency_test"},
```

Comment:
This flag name is confusing; it's unclear whether it's an int or a string. Suggest "max_publish_latency". However, I think the bigger problem is that this flag is not at the same layer of abstraction as the other flags above (such as number_of_messages). Is it sensible to specify both number_of_messages and max_publish_latency_test? I'm assuming 'yes', since the former specifies the stopping condition for an individual "run", whereas the latter specifies the stopping condition for a suite of such runs. So suggest breaking this up into two separate concepts: one which captures the aforementioned "suite", and one which captures a single iteration. IIUC, most existing flags apply to the "iteration". Then we each concept can be captured in its own class with its own flags.